src/lib.rs

```diff
@@ -221,7 +221,7 @@ pub enum UuidVariant {
 #[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
 pub struct Uuid {
     /// The 128-bit number stored in 16 bytes
-    bytes: UuidBytes,
+    pub bytes: UuidBytes,
```

Comment:
Instead of making the `bytes` public, make a struct with the same layout in the macro crate

Reply:
I have made it private。and use this instead 
```rust 
pub const fn new(bytes:[u8;16]) -> Uuid {
    Uuid{bytes}
}
```